examples/mlperf/initializers.py

```diff
@@ -41,7 +41,8 @@ def __init__(self, in_features, out_features, bias=True, std=0.02):
     self.bias = Tensor.zeros(out_features, dtype=dtypes.float32) if bias else None
   
   def __call__(self, x:Tensor):
-    return x.linear(self.weight.cast(dtypes.default_float).transpose(), self.bias.cast(dtypes.default_float) if self.bias is not None else None)
+    x = x.contiguous().cast(dtypes.half if getenv("HALF_LINEAR", 0) else dtypes.default_float)
```

Comment:
can you make a variable for `dtypes.half if getenv("HALF_LINEAR", 0) else dtypes.default_float`? i think that reads better and less repeated code